src/Tests/Moryx.Communication.Sockets.IntegrationTests/CommunicationSocketsTests.cs

```diff
@@ -9,6 +9,10 @@
 
 namespace Moryx.Communication.Sockets.IntegrationTests
 {
+    // preprocessor statement for differentiating between different os
```

Comment:
Please also reference the issue in the Description of this PR, github will then automatically lilnk the PR and the issue 😊